pontoon/base/views.py

```diff
@@ -116,31 +168,51 @@ def locales(request):
     return render(request, 'locales.html', data)
 
 
-def project(request, slug, template='project.html'):
-    """Project view."""
-    log.debug("Project view.")
+class ProjectView(TemplateModelView):
+    """
+    Renders a page for the project.
```

Comment:
This docstring doesn't tell us anything more than the classname did, no need to include it.

Also my comments about not changing the locale view apply here as well.